src/_pytest/monkeypatch.py

```diff
@@ -325,18 +324,15 @@ def syspath_prepend(self, path) -> None:
 
         invalidate_caches()
 
-    def chdir(self, path) -> None:
+    def chdir(self, path: Union[str, "os.PathLike[str]"]) -> None:
         """Change the current working directory to the specified path.
 
-        Path can be a string or a py.path.local object.
+        Path can be a string or a path object.
         """
         if self._cwd is None:
             self._cwd = os.getcwd()
         if hasattr(path, "chdir"):
```

Comment:
We might replace this with if/else with:

```python
Path(path).chdir()
```

Reply:
Looks like `Path` doesn't have a `chdir()` method. So I just went with `os.chdir(path)` instead (i.e. removed the hasattr stuff). If something out there somehow needed that, I guess we'd find out :)

Reply:
Oh I could swear it did. 😁 

Yeah I think `os.chdir(path)` there is the right call. 👍